spec/queries/bp_measures_query_spec.rb

```diff
@@ -27,9 +27,11 @@ def ist_zone
     facility = create(:facility)
     end_of_jan = ist_zone.parse("January 31st 23:59:59 IST")
     beg_of_feb = ist_zone.parse("February 1st 00:00:00 IST")
+
     create(:blood_pressure, facility: facility, recorded_at: end_of_jan, user: user_1)
     create(:blood_pressure, facility: facility, recorded_at: beg_of_feb, user: user_2)
-    Time.use_zone(ist_zone) do
+
+    with_reporting_time_zone do
```

Comment:
This is the fix - we weren't using Groupdate for the query under test, so things would get sorted as UTC instead of IST.